### PR TITLE
get ios dta working on 3.2

### PR DESCRIPTION
**Pull request by aruniverse**
uses itwin/mobile-native-ios w/ xcframework
only snapshots work

### CODE REVIEW

File: test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -425,6 +427,25 @@
...
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		F4A4D85E28A599050015ED63 /* XCRemoteSwiftPackageReference "mobile-native-ios" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/iTwin/mobile-native-ios";
+			requirement = {
+				kind = exactVersion;
+				version = 3.2.14;
```
Comment (pmconne) [inline test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app.xcodeproj/project.pbxproj:437]: Will this need to be updated each time we publish a new addon? If so please update the two [wiki articles](https://dev.azure.com/bentleycs/iModelTechnologies/_wiki/wikis/iModelTechnologies.wiki/2036/Creating-a-new-imodel.js-addon).

Comment (aruniverse) [reply test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app.xcodeproj/project.pbxproj:437]: thx, updated

**Review by pmconne — APPROVED** [review]

File: core/common/package.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
   "typings": "lib/cjs/core-common",
   "license": "MIT",
   "scripts": {
-    "build": "npm run -s build:cjs",
-    "build:ci": "npm run -s build && npm run -s build:esm",
+    "build": "npm run -s build:cjs && npm run -s build:esm",
```
Comment (calebmshafer) [inline core/common/package.json:10]: Should we be doing this for all of the packages again? Seems like we're selectively adding some of these back into the main build process.

Comment (aruniverse) [reply core/common/package.json:10]: I added this in my webpack5-continued pr in 3.3; this is to get around developers having to manually build the esm version of core-common which is needed when they go to build their test apps.

should be unecessary once steves pr to decouple core-common with the rpc pr goes through

**Comment by aruniverse** [comment]
@mergifiyio backport master

**Pull request merged**